spec/controllers/interactive_page_controller_spec.rb

```diff
@@ -186,11 +186,11 @@
 
     it 'does not show page areas which are not selected to be shown' do
```

Comment:
Maybe we can use the it title from line 164 (`calls LARA.addSidebar content on pages which have authored sidebar`
to here, and then delete that spec It seems redundant because this one tests both cases.

Also It wasn't obvious to me at first that the Page factory was creating pages with sidebars by default.